recipes/xsimd/all/conanfile.py

```diff
@@ -56,6 +56,12 @@ def validate(self):
         if Version(self.version) < "8.0.0" and is_apple_os(self) and self.settings.arch in ["armv8", "armv8_32", "armv8.3"]:
             raise ConanInvalidConfiguration(f"{self.ref} doesn't support macOS M1")
 
+        # TODO: There are compilation errors on apple-clang/13.0.0 with cppstd=17
+        if self.settings.compiler == "apple-clang" and \
+            Version(self.settings.compiler.version).major <= "13" and \
+            self.settings.compiler.get_safe("cppstd") in ["17", "gnu17", "20", "gnu20",]:
```

Comment:
We'll make sure to clean this up once Conan 1 is no longer compiled in the pipeline :)